ext/standard/basic_functions.c

```diff
@@ -1427,7 +1429,11 @@ PHPAPI int _php_error_log_ex(int opt_err, const char *message, size_t message_le
 			break;
 
 		default:
+#ifdef HAVE_SYSLOG_H
 			php_log_err_with_severity(message, LOG_NOTICE);
+#else
+			php_log_err_with_severity(message, 5);
```

Comment:
Magic numbers are not a good idea. If WASM doesn't define LOG_NOTICE, then have a block somewhere else that defines this (and I suspect other constants) in one go.